service/src/main/kotlin/app/cash/backfila/service/persistence/DbEventLog.kt

```diff
@@ -0,0 +1,68 @@
+package app.cash.backfila.service.persistence
+
+import java.time.Instant
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.Table
+import misk.hibernate.DbTimestampedEntity
+import misk.hibernate.DbUnsharded
+import misk.hibernate.Id
+
+@Entity
+@Table(name = "event_logs")
+class DbEventLog() : DbUnsharded<DbEventLog>, DbTimestampedEntity {
+
+  enum class Type {
+    STATE_CHANGE,
+    CONFIG_CHANGE,
+    ERROR
+  }
+
+  @javax.persistence.Id
+  @GeneratedValue
+  override lateinit var id: Id<DbEventLog>
+
+  @Column
+  override lateinit var created_at: Instant
+
+  @Column
+  override lateinit var updated_at: Instant
+
+  @Column(nullable = false)
+  lateinit var backfill_run_id: Id<DbBackfillRun>
+
+  @Column
+  var partition_id: Id<DbRunPartition>? = null
+
+  /** User that triggered the event, null when the event was triggered by the service. */
+  @Column
+  var user: String? = null
```

Comment:
= 'backfila' ?

Reply:
Blank user seems pretty clear it wasn't done by a user

Reply:
Sure... sgtm